src/spotify.cpp

```diff
@@ -64,8 +64,13 @@ void Spotify::connect() {
     // start polling
     m_pollingTimer->start();
 
-    // if it's the first startup, connect signals
+    // if it's the first startup, add available entity
     if (m_startup) {
+        EntityInterface* entity = static_cast<EntityInterface*>(m_entities->getEntityInterface(m_entityId));
+        if (entity) {
+            addAvailableEntity(m_entityId, entity->type(), integrationId(), entity->friendly_name(),
+                               entity->supported_features());
```

Comment:
Build is failing due to unknown method `addAvailableEntity`!
This needs to be included in a separate pull request once the integrations.library is merged and released.
Please revert commit and create a new PR which references the new integrations.library version in `dependencies.cfg`